README.md

```diff
@@ -16,43 +16,42 @@ npm install --save-dev ovh-ui-kit-bs
 
 ## Usage
 
-### Dependencies
-
-Those dependencies will be installed automatically by npm:
-
-- [bootstrap](https://github.com/twbs/bootstrap)
-- [less-plugin-remcalc](https://github.com/ovh-ux/less-plugin-remcalc)
-- [ovh-ui-kit](https://github.com/ovh-ux/ovh-ui-kit)
-
 ### Integration
 
 #### html
 
 ```html
-<link rel="stylesheet" href="./node_modules/ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css">
+<link rel="stylesheet" href="./node_modules/ovh-ui-kit-bs/dist/ovh-ui-kit-bs.min.css">
 ```
 
 #### less
 
-[less-plugin-remcalc](https://github.com/ovh-ux/less-plugin-remcalc) is needed to compile the LESS import.
+Those dependencies are needed to compile the LESS import.
+
+- [bootstrap](https://github.com/twbs/bootstrap)
```

Comment:
capital **B** please :)

Reply:
For `Bootstrap` ? I don't know...
It was to keep a coherence by using dependencies name for all :)